databases/installing-postgresql.md

```diff
@@ -127,5 +127,10 @@ BigSQL currently installs pgAdmin version 3 which is deprecated. It's best to ch
 
 *PostgreSQL 10 has been installed in /opt/PostgreSQL-10 directory.*
 
+## Problems
+
+**Common issues users get while installing:**
+* **Using non english characters in password** - Avoid using it.
```

Comment:
Is their any restriction on using passwords other than English characters ?

Reply:
As found on stackoverflow. Many users find trouble while re-entering passwords other than english characters. It doesn't match sometimes. So its better to avoid it.

Reply:
Oh, okay.